src/v0/util/facebookUtils/networkHandler.js

```diff
@@ -100,12 +100,16 @@ const errorDetailsMap = {
   190: {
     460: new ErrorDetailsExtractorBuilder()
       .setStatus(400)
+      .setStat('accessTokenExpired')
```

Comment:
```suggestion
      .setStat({
        [tags.TAG_NAMES.META]: tags.ERROR_TYPES.UNAUTHORISED,
      })
```
Please check spelling and make necessary changes